internal/git/config_test.go

```diff
@@ -1,8 +1,9 @@
-package release
+package git_test
```

Comment:
Any particular reason why this package should have the `_test` suffix?

Reply:
Ah yes! Sorry should've left a PR comment, it was getting an import cycle otherwise:

```
# github.com/goreleaser/goreleaser/internal/git
import cycle not allowed in test
package github.com/goreleaser/goreleaser/internal/git (test)
	imports github.com/goreleaser/goreleaser/internal/testlib
	imports github.com/goreleaser/goreleaser/internal/git
FAIL	github.com/goreleaser/goreleaser/internal/git [setup failed]
```

I can undo the change to move the git config handling into this package so its re-usable, but figured the code made sense in this one spot.

Reply:
Gotcha, I guess it's fine - we can do the refactoring in a separate PR.